utils/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+@Suppress("DSL_SCOPE_VIOLATION") // TODO: Remove once KTIJ-19369 is fixed
+plugins {
+    id("java-library")
```

Comment:
why java-library? also you can get rid of `@Suppress("DSL_SCOPE_VIOLATION")`  using plugin. google it. also, should be fixed in new versions